deploy/service_account.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
Similarly I don't think this file is being used either.

Reply:
The deploy/service_account.yaml file has been deleted.